graph/src/data/query/error.rs

```diff
@@ -78,10 +105,46 @@ impl fmt::Display for QueryExecutionError {
                 f,
                 "Only a single top-level field is allowed in subscriptions"
             ),
+            QueryExecutionError::SubgraphParseError(s) => {
+                write!(f, "Failed to get subgraph ID from type: {}", s)
+            }
+            QueryExecutionError::BuildRangeTypeError => {
+                write!(f, "Range inputs must be an integer")
+            }
+            QueryExecutionError::BuildFilterError => write!(f, "Filter must by an object"),
+            QueryExecutionError::EntityAttributeError => {
+                write!(f, "Attribute does not belong to entity")
+            }
+            QueryExecutionError::ListTypesError() => {
+                write!(f, "List field contains inconsistent Value types")
+            }
+            QueryExecutionError::ListFilterError => write!(
+                f,
+                "Non-list value resolved for list filter  \
+                 *Hint*: IN and NOT_IN filters take an array of values"
+            ),
         }
     }
 }
 
+impl From<QueryExecutionError> for Vec<QueryExecutionError> {
+    fn from(e: QueryExecutionError) -> Self {
+        vec![e]
+    }
+}
+
+impl From<FromHexError> for QueryExecutionError {
+    fn from(_e: FromHexError) -> Self {
+        QueryExecutionError::NamedTypeError("Bytes".to_string())
```

Comment:
This is not correct. `NamedTypeError` is used for when a type cannot be resolved. This is a case of an invalid value provided for type `Bytes`.